cmd/k8s-operator/ingress-for-pg.go

```diff
@@ -473,16 +486,22 @@ func (r *HAIngressReconciler) maybeCleanup(ctx context.Context, hostname string,
 	if err != nil {
 		return false, fmt.Errorf("error deleting VIPService: %w", err)
 	}
+
+	// 3. Clean up any cluster resources
```

Comment:
We can't do owner refs because cert Secret and RBAC are in tailscale namespace, but Ingress is not